src/AWS.Messaging/Configuration/SQSMessagePollerConfiguration.cs

```diff
@@ -87,6 +88,11 @@ internal class SQSMessagePollerConfiguration : IMessagePollerConfiguration
     /// </remarks>
     public int WaitTimeSeconds { get; init; } = DEFAULT_WAIT_TIME_SECONDS;
 
+    /// <summary>
+    /// Function that that indicated whether SQS poller should stop polling or continue;
+    /// </summary>
+    public Func<AmazonSQSException, bool> FatalErrorChecker { get; set; } = IsSQSExceptionFatal;
```

Comment:
Suggestion to clean up documentation and naming:
```suggestion
    /// Function that indicates whether an <see cref="AmazonSQSException"/> should stop the poller or continue.
    /// </summary>
    public Func<AmazonSQSException, bool> IsSQSExceptionFatal { get; set; } = IsSQSExceptionFatal;
```